### ISSUE TITLE
2.1 Snapshot doesn't include the latest version of the logstash-input-beats.

### ISSUE THREAD

**Issue by ph** [elastic/logstash#4235]
2.0.0 - November 17, 2015 is the latest version (see https://rubygems.org/gems/logstash-input-beats) but it require concurrent-ruby 0.9.1, and logstash-core need 0.9.2 and 0.9.4 is the only version that can meet the dependency requirements because it was more relaxed, oh well versioning :japanese_ogre: 

2.1 lock  file
```
    logstash-input-beats (0.9.4)
      concurrent-ruby
      logstash-codec-plain
      logstash-core (>= 1.5.4, < 3.0.0)
```

from https://github.com/elastic/logstash/blob/2.1/Gemfile.jruby-1.9.lock#L249-L252

This should be fixed by https://github.com/logstash-plugins/logstash-input-beats/pull/30 which will relax the dependency.

I think it would be best for us, to be strict on the core and the related gems and be flexible on the library, WDYT?

**Comment by ph** [comment]
@colin @purbon input on this?

**Comment by purbon** [comment]
Yeah, I understand you pain here. to be strict on plugins is hard and makes you have troubles like this I guess. As we did with ruby-maven, being strict on core is good or just be strict everywhere, but we should prevent this two different needs. makes sense?

**Comment by ph** [comment]
released beats 2.0.2 with an updated dependency on concurrent-ruby.

**Comment by colinsurprenant** [comment]
:thumbsup:

**Comment by jsvd** [comment]
logstash-input-beats 2.0.3 is now included in 2.1 branch https://github.com/elastic/logstash/blob/2.1/Gemfile.jruby-1.9.lock#L249-L252
